W-22103685-graphQI-introspection (#1042)
mulesoft/docs-gateway: gateway-home/modules/ROOT/pages/index.adoc

| External Processing ^| Sends the incoming HTTP requests or outgoing HTTP responses to an external gRPC service for additional processing ^| xref:gateway::policies-included-external-processing.adoc[Yes] ^| xref:gateway::policies-included-external-processing.adoc[Yes] ^| xref:gateway::policies-included-external-processing.adoc[Yes] ^| No 

> | GraphQL Introspection Control ^| Lets you allow or block the GraphQL introspection meta-fields `__schema`, `__type`, and `__typename` ^| xref:gateway::policies-included-graphql-introspection-control.adoc[Yes] ^| xref:gateway::policies-included-graphql-introspection-control.adoc[Yes] ^| xref:gateway::policies-included-graphql-introspection-control.adoc[Yes] ^| No 

Reviewer: ```suggestion
| GraphQL Introspection Control ^| Blocks or allows requests that  access the `__schema`, `__type`, and `__typename` GraphQL meta fields ^| xref:gateway::policies-included-graphql-introspection-control.adoc[Yes] ^| xref:gateway::policies-included-graphql-introspection-control.adoc[Yes] ^| xref:gateway::policies-included-graphql-introspection-control.adoc[Yes] ^| No 
```

Reviewer: use this everywhere
